### ISSUE TITLE
MuseScore 4 icon is pixelated

### ISSUE THREAD

**Issue by aaronsendra** [musescore/MuseScore#18489]
### Issue type

Choose option...

### Bug description

MuseScore icon view is pixelated on Linux, while in older versions it was good, as you can see on the screenshot.

### Steps to reproduce

1. Download MuseScore AppImage.
2. See the pixelated icon on Dolphin or the desktop.

### Screenshots/Screen recordings

![Screenshot](https://github.com/musescore/MuseScore/assets/46270385/20be9b74-fe22-4153-8058-edd8e2ce3847)

### MuseScore Version

4.1.0.231900501

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

LliureX 21 (Ubuntu 20.04.6, KDE 5.25.2)

**Comment by MarcSabatella** [comment]
How did you install the 4.1?  Normally (running from command line with "install" option) it should have installed the various different size icons into your applications folder, then it is up to your window manager to choose the right one to display.  What is the folder that your screenshot is showing?  That doesn't seem to be the normal install location.  Also the install location should have created more 'human friendly" command names.

**Comment by aaronsendra** [comment]
I didn't install it, I downloaded the AppImage for Linux. It is the desktop folder. The icon of the 4.0.2 is shown properly, while the 4.1 one is pixelated.

**Comment by MarcSabatella** [comment]
Maybe I'm misunderstanding something, but as far as I know, until you install it, there is no icon to display.  I guess maybe your particular file manager might be employing some special techniques to attempt to read the icon directly from within the AppImage file, but I don't know that this is actually supported.  s far as I know, most file managers won't display an icon at all for an AppImage file that isn't installed properly (with a corresponding ".desktop" file).  The correct way to get an icon - and everything else associated with an installed app - is to simply install it normally via the "install" option.

Looks like you are using Dolphin?  I just tried that using Dolphin on Debian and got no icon at all for any version of MuseScore when looking at the AppImage itself in my own folder, just as I expect.  Only the installed programs show with icons.  Maybe you or your distribution did some custom configuration to get it show something at all?

**Comment by aaronsendra** [comment]
As far as I know, an AppImage is a portable version of a program for Linux. To open it, I make it executable and then run it, but it does not install anywhere. But maybe I have been using AppImage files on a wrong way. The problem is that for 4.0.2 the icon is properly displayed, while in 4.1 it looks blurred

**Comment by MarcSabatella** [comment]
Yes, some AppImages don't support installation.  But MuseScore does, and that's how it is intended to be used.  So simply run the program from the command line with the "install" option to install it.  This creates the ".desktop" file and places the proper icons in the proper locations on your system.

Again, if you don't have a valid ".desktop" file, then your AppImage isn't installed and your OS won't know how to make file associations, add it to application menus, display program icons, or display file icons for your scores - those are all the job of the ".desktop" file.  Without the desktop file, your file manager is just guessing at what icon to display.  I'm surprised it ever worked.  I guess whatever magic Dolphin is using on your system to attempt to guess at an appropriate icon was depending on something within the AppImage that is now being done differently, and maybe there is a way to change that to make that magic work again.  But as I have explained, this magic this is *not* the generally supported way to get icons on Linux.  If you want icons, you are supposed to install normally, with ".desktop" file.  Some AppImages might make you do that manually, but MuseScore doesn't - it provides that functionality itself, all you need to do is tell it to do so.
